.github/workflows/windows-x86.yml

```diff
@@ -0,0 +1,92 @@
+name: windows 32bit
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-2019
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Install swig
+        run: choco install swig
+
+      - name: Install Qt and Jom
+        run: |
+          pip install aqtinstall
+          python -m aqt install 5.13.1 windows desktop win32_msvc2017
+          wget http://download.qt.io/official_releases/jom/jom.zip -OutFile jom.zip
```

Comment:
Looks like there's no integrity protection for this (not even https)?

Could we pull these dependencies through Artifactory or some such somehow?